packages/sources/react/src/components/indicators/VtmnProgressbar/VtmnProgressbar.tsx

```diff
@@ -149,11 +149,11 @@ export const VtmnProgressbar = ({
               y1="50%"
               y2="50%"
               style={
-                isDeterminate
-                  ? {
-                      transform: `translateX(${progress - 100}%)`,
-                    }
-                  : {}
+                {
+                  '--vtmn-progressbar_progress-transform': isDeterminate
+                    ? `translateX(${progress - 100}%)`
+                    : 'unset',
```

Comment:
What do you think about this short solution ?

```html
style={isDeterminate && `translateX(${progress - 100}%)`}
```

Reply:
Do you mean
```
style={isDeterminate && {
    '--vtmn-progressbar_progress-transform': `translateX(${progress - 100}%)`
}}

```
?

I agree that this way the default value is managed by the css directly

Reply:
Why do we keep SVG for the linear progress? Seems to me to be too much complexity instead of using a `div` tag and a hidden `progress` for a11y.

Reply:
Hi @romuleald, thanks for your comment!
SVG doesn't bring to much complexity here, after all it is juste an `svg` tag with a `line` tag, not that different from having a `div` inside a `div`. At the end of the day I agree that simpler shapes should be drawn with CSS instead of SVG though. In both cases the line/div is animated with a `transform` property cached in a CSS variable, complexity remains the same.

Finally, note that SVG is kept here to keep the consistency between linear progress and circular progress (this one is easier to draw with SVG) and, above all, the purpose of this issue is only to fix a minor change. I agree with the `progress` pattern, probably the direction we'll take when the component is reworked in the v1.